app/src/main/java/com/eveningoutpost/dexdrip/NFCReaderX.java

```diff
@@ -218,10 +219,32 @@ public static boolean HandleGoodReading(String tagId, byte[] data1, final long C
         return HandleGoodReading(tagId, data1, CaptureDateTime, false, null, null);
     }
 
+
+    public static void SendLibrereading(final String tagId, byte[] data1, final long CaptureDateTime, byte []patchUid,  byte []patchInfo){
+        if(!Home.get_master()) {
+            return;
+        }
```

Comment:
Can we add a check for the librewholehouse preference switch here and return if it isn't set?

Reply:
This information is needed for all followers (not related to all house coverage).
It allows them to have per minute information (libre trend), sensor life and battery life.
Even Followers who will not use all house need this information.